src/libraries/System.IO.FileSystem/tests/Enumeration/AttributeTests.cs

```diff
@@ -263,6 +263,8 @@ public void IsReadOnlyAttribute()
             };
 
             Assert.Equal(new string[] { fileTwo.FullName }, enumerable);
```

Comment:
Line 247, I wonder if it could be changed to `using TestDirectory testDirectory = new TestDirectory();`. I don't remember if there were any restrictions in the current context.

I mention it because I don't see the point of restoring the file attributes if the file is supposed to get deleted at the end.